source/developers-guide/models/index.md

```diff
@@ -89,7 +89,6 @@ When a model object is first added to the database, the `prePersist` and `postPe
 namespace SwagModel\Subscriber;
 
 use Doctrine\Common\EventSubscriber;
```

Comment:
we think that our example is correct, as we are using the `Doctrine\Common\EventSubscriber` and not the Shopware event subscriber. 
maybe you can add another example with the Shopware Eventsubscriber?

Reply:
Sure. But I think there is no need, isn't it?